mantle/kola/tests/util/luks.go

```diff
@@ -79,3 +80,26 @@ func LUKSSanityTest(c cluster.TestCluster, tangd TangServer, m platform.Machine,
 	luksDump = c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
 	mustMatch(c, "Cipher: *aes", luksDump)
 }
+
+// LUKSSanityCEXTest verifies that the rootfs is encrypted with Cex based LUKS
+func LUKSSanityCEXTest(c cluster.TestCluster, m platform.Machine, rootPart string) {
+	var err error
+	luksDump := c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
+	mustMatch(c, "cipher: paes-*", luksDump)
+	mustNotMatch(c, "Cipher: *cipher_null-ecb", luksDump)
+	mustMatch(c, "0: paes-verification-pattern", luksDump)
+	mustNotMatch(c, "9: *coreos", luksDump)
+
+	err = m.Reboot()
+
+	if err != nil {
+		c.Fatalf("Failed to reboot the machine: %v", err)
+	}
+	luksDump = c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
+	mustMatch(c, "cipher: paes-*", luksDump)
+
+	err = coretest.TestRHCOSGrowfs()
```

Comment:
Call this before the reboot.

Reply:
I tried by adding the TestRHCOSGrowfs() before reboot but same issue.